src/configs.rs

```diff
@@ -29,6 +29,9 @@ pub(crate) enum SubCommand {
 
 #[derive(Parser, Debug, Clone)]
 pub(crate) struct RunArgs {
+    /// AWS API Endpoint
+    #[clap(long, default_value = "")]
+    pub endpoint: String,
```

Comment:
Consider use `Option` to avoid the `.is_empty()` check, please.

Reply:
please check it works for you now